modules/authentication/proc-enabling-authentication-with-rhbk.adoc

```diff
@@ -45,7 +45,9 @@ auth:
         metadataUrl: ${AUTH_OIDC_METADATA_URL}
         clientId: ${AUTH_OIDC_CLIENT_ID}
         clientSecret: ${AUTH_OIDC_CLIENT_SECRET}
+        prompt: auto #add this config
 signInPage: oidc
+dangerouslyAllowSignInWithoutUserInCatalog: true
```

Comment:
Please remove this field as it is not mandatory (and also not safe for production)